solidity/contracts/ECDSARewards.sol

```diff
@@ -0,0 +1,127 @@
+/**
+▓▓▌ ▓▓ ▐▓▓ ▓▓▓▓▓▓▓▓▓▓▌▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▄
+▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▌▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
+  ▓▓▓▓▓▓    ▓▓▓▓▓▓▓▀    ▐▓▓▓▓▓▓    ▐▓▓▓▓▓   ▓▓▓▓▓▓     ▓▓▓▓▓   ▐▓▓▓▓▓▌   ▐▓▓▓▓▓▓
+  ▓▓▓▓▓▓▄▄▓▓▓▓▓▓▓▀      ▐▓▓▓▓▓▓▄▄▄▄         ▓▓▓▓▓▓▄▄▄▄         ▐▓▓▓▓▓▌   ▐▓▓▓▓▓▓
+  ▓▓▓▓▓▓▓▓▓▓▓▓▓▀        ▐▓▓▓▓▓▓▓▓▓▓▌        ▓▓▓▓▓▓▓▓▓▓▌        ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
+  ▓▓▓▓▓▓▀▀▓▓▓▓▓▓▄       ▐▓▓▓▓▓▓▀▀▀▀         ▓▓▓▓▓▓▀▀▀▀         ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▀
+  ▓▓▓▓▓▓   ▀▓▓▓▓▓▓▄     ▐▓▓▓▓▓▓     ▓▓▓▓▓   ▓▓▓▓▓▓     ▓▓▓▓▓   ▐▓▓▓▓▓▌
+▓▓▓▓▓▓▓▓▓▓ █▓▓▓▓▓▓▓▓▓ ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓  ▓▓▓▓▓▓▓▓▓▓
+▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓ ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓  ▓▓▓▓▓▓▓▓▓▓
+
+                           Trust math, not hardware.
+*/
+
+pragma solidity 0.5.17;
+
+import "@keep-network/keep-core/contracts/Rewards.sol";
+import "./BondedECDSAKeepFactory.sol";
+import "./BondedECDSAKeep.sol";
+
+/// @title KEEP Random ECDSA Signer Subsidy Rewards for the Sep 2020 release.
+/// @notice Contract distributes KEEP rewards to signers that were part of
+/// the keeps which were created by the BondedECDSAKeepFactory contract:
+/// https://etherscan.io/address/0xA7d9E842EFB252389d613dA88EDa3731512e40bD
+///
+/// The amount of KEEP to be distributed is determined by funding the contract,
+/// and additional KEEP can be added at any time.
+///
+/// When an interval is over, it will be allocated a percentage of the remaining
+/// unallocated rewards based on its weight, and adjusted by the number of keeps
+/// created in the interval if the quota is not met.
+///
+/// The adjustment for not meeting the keep quota is a percentage that equals
+/// the percentage of the quota that was met; if the number of keeps created is
+/// 80% of the quota then 80% of the base reward will be allocated for the
+/// interval.
+///
+/// Any unallocated rewards will stay in the unallocated rewards pool,
+/// to be allocated for future intervals. Intervals past the initially defined
+/// schedule have a weight of the last scheduled interval.
+///
+/// Keeps can receive rewards once the interval they were created in is over,
+/// and the keep has been marked as closed.
+/// There is no time limit to receiving rewards, nor is there need to wait for
+/// all keeps from the interval to be marked as closed.
+/// Calling `receiveReward` automatically allocates the rewards for the interval
+/// the specified keep was created in and all previous intervals.
+///
+/// If a keep is terminated, that fact can be reported to the reward contract.
+/// Reporting a terminated keep returns its allocated reward to the pool of
+/// unallocated rewards.
+contract ECDSARewards is Rewards {
+    // BondedECDSAKeepFactory deployment date, Sep-14-2020 interval started.
+    // https://etherscan.io/address/0xA7d9E842EFB252389d613dA88EDa3731512e40bD
+    uint256 internal constant ecdsaFirstIntervalStart = 1600087297;
```

Comment:
Can we set it to 12 AM 2020-09-14 for consistency? That's what we do with all timestamps in other reward contracts.